examples/esp32-with-cred/esp32-server-with-cred.rs

```diff
@@ -136,7 +136,7 @@ fn main() -> Result<(), ServerError> {
         RobotRepresentation::WithRobot(LocalRobot::new(res))
     };
     #[cfg(not(feature = "qemu"))]
-    let repr = RobotRepresentation::WithRegistry(ComponentRegistry::default());
+    let repr = RobotRepresentation::WithRegistry(Box::new(ComponentRegistry::default()));
```

Comment:
What motivated this change in this review?

Reply:
The size difference between the two enum was pretty important so for more made sense the put it in a Box. Saves us some stack too!